com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -2185,264 +2187,276 @@ void PreprocessLightData(ref ProcessedLightData processedData, VisibleLight ligh
         // This will go through the list of all visible light and do two main things:
         // - Precompute data that will be reused through the light loop
         // - Discard all lights considered unnecessary (too far away, explicitly discarded by type, ...)
-        int PreprocessVisibleLights(HDCamera hdCamera, CullingResults cullResults, DebugDisplaySettings debugDisplaySettings, in AOVRequestData aovRequest)
+        int PreprocessVisibleLights(CommandBuffer cmd, HDCamera hdCamera, in CullingResults cullResults, DebugDisplaySettings debugDisplaySettings, in AOVRequestData aovRequest)
```

Comment:
I assumed most changes in here are mostly tabs + trivial passing of args to the changed functions, if there is something else to need proper review do let me know

Reply:
top right settings of the diff, ensure to remove white spaces